zeroex/ordervalidator/order_validator.go

```diff
@@ -354,10 +355,24 @@ func (o *OrderValidator) BatchValidate(ctx context.Context, rawSignedOrders []*z
 					d := b.Duration()
 					if d == maxDuration {
 						<-semaphoreChan
-						log.WithFields(log.Fields{
-							"error":     err.Error(),
-							"numOrders": len(trimmedOrders),
-						}).Warning("Gave up on GetOrderRelevantStates request after backoff limit reached")
+						var fields log.Fields
+						match, err := regexp.MatchString("abi: improperly formatted output", err.Error())
```

Comment:
This shadows the definition for `err`, which will break the line of code below `if err.Error() == "VM execution error."`. How about renaming it to `regexErr`?

```suggestion
						match, regexErr := regexp.MatchString("abi: improperly formatted output", err.Error())
```